src/Stratiteq.Identity.Client.AAD/AADAppConfiguration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Stratiteq Sweden AB. All rights reserved.
+//
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Extensions.Configuration;
+
+namespace Stratiteq.Identity.Client.AAD
+{
+    /// <summary>
+    /// Contains the information needed to make authenticated requests to a web API protected with Azure Active Directory (and role based authentication).
+    /// </summary>
+    public sealed class AADAppConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AADAppConfiguration"/> class.
+        /// </summary>
+        /// <param name="configuration">The configuration.</param>
+        public AADAppConfiguration(IConfiguration configuration)
```

Comment:
To increase flexibility, there should be an additional ctor that takes the values directly (tenantId, clientId etc)

Reply:
Agree! I was considering it but never got to it!